acme/acme/standalone.py

```diff
@@ -262,7 +263,7 @@ def simple_tls_sni_01_server(cli_args, forever=True):
 
     certs = {}
 
-    _, hosts, _ = next(os.walk('.'))
+    _, hosts, _ = next(os.walk('.')) # type: ignore # https://github.com/python/mypy/issues/465
```

Comment:
yep. their release schedule seems pretty erratic though, so leaving this for now.

Reply:
`mypy` did a release yesterday fixing this! Can we update `tools/dev_constraints.txt` to `mypy==0.600` and remove this ignore?

Reply:
I just realized we may need to be careful here. From the `typing` description on PyPI:

> NOTE: in Python 3.5 and later, the typing module lives in the stdlib, and installing this package has NO EFFECT. To get a newer version of the typing module in Python 3.5 or later, you have to upgrade to a newer Python (bugfix) version. For example, typing in Python 3.6.0 is missing the definition of ‘Type’ – upgrading to 3.6.2 will fix this.

Even though this fix is included in the latest version of `typing`, it may not be included in every version of Python 3.5+ installed by devs from their OS package manager. If we think this is the case, we should probably leave this ignore. Hopefully we don't hit too many issues like this where we have to work around old versions of `typing` to allow testing to work consistently in Travis and different dev environments.

Reply:
since this fix hasn't even made it into python bugfix versions yet, leaving this for now. we can decide what we want to do about this later.